login-Signup.css

```diff
@@ -0,0 +1,237 @@
+* {
+    box-sizing: border-box;
+}
+
+@media (min-width: 375px) {
+
+    /* 버튼 관련 */
+    .buttons {
+        width: 100%;
+        height: 56px;
+        border-radius: 40px;
+        background-color: #9CA3AF;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        color: #F3F4F6;
+        margin-bottom: 24px;
+        font-size: 20px;
+        font-weight: 600;
+        border: 0px;
+    }
+
+    /* 전체적인 틀 */
+    body {
+        font-family: "Pretendard Variable", Pretendard, sans-serif;
+
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+    }
+
+    container {
+        margin-top: 80px;
+        width: 100%;
+        max-width: 400px;
+        height: auto;
+        margin-right: 16px;
+        margin-left: 16px;
+    }
+
+    /* 로고관련 */
+    #logo > a {
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        gap: 11px;
+        margin-bottom: 24px;
+    }
+
+    #logo img:first-child {
+        width: 52px;
+    }
+
+    #logo img:last-child {
+        width: 133px;
+    }
+
+    /* 폼테그 관련 */
+    form {
+        display: flex;
+        flex-direction: column;
+        color: #1F2937;
+        font-size: 18px;
+        font-weight: 700;
+    }
+
+    form > label {
+        margin-bottom: 8px;
+    }
+
+    form > input {
+        width: 100%;
```

Comment:
❗️ 수정요청
인풋에 값을 입력시 너무 좌우에 딱 붙어서 입력이되네요!
디자인을 확인해보시고 적절한 padding값을 추가해주세요~

<img width="441" alt="스크린샷 2025-03-27 오전 11 40 42" src="https://github.com/user-attachments/assets/926b0977-2d85-41cc-b311-a27ef86e3320" />

```suggestion
    form > input {
        padding: 0 16px;
        width: 100%;
```